### ISSUE TITLE
Add an assumptions section?

### ISSUE THREAD

**Issue by AdamJLemmon** [mattrglobal/oidc-client-bound-assertions-spec#50]
Is it worth adding a small section on assumptions?  

ie. noting that the holder knows what the credential issuer offers and that the CI offers just one credential?  They're asking for default credential from the CI. ... more complex request syntax out of scope?

**Comment by tplooker** [comment]
Agree we need to find a place to make this rationale available, specifications don't tend to be the best place to do this though, lets explore some other options?

**Comment by AdamJLemmon** [comment]
Yes agreed sounds good to me. Feeling like getting these decisions recorded somewhere will be useful though.
